apps/lead/models.py

```diff
@@ -250,6 +250,10 @@ def get_associated_entities(cls, project_id, lead_ids):
             'assessments': Assessment.objects.filter(lead__project_id=project_id, lead__in=lead_ids).count(),
         }
 
+    @property
+    def verified_entries_count(self):
+        return self.entry_set.filter(verified=True).count()
+
```

Comment:
This is okay but we should use annotate to calculate the count on the listing view.